docker-compose.yml

```diff
@@ -7,20 +7,34 @@ services:
     ports:
       - "8000:8000"
     environment:
+      - UPLOAD_FOLDER=/var/uploads
+      - LOG_FOLDER=/var/log/CTFd
       - DATABASE_URL=mysql+pymysql://root:ctfd@db/ctfd
     volumes:
-      - .data/CTFd/logs:/opt/CTFd/CTFd/logs
-      - .data/CTFd/uploads:/opt/CTFd/CTFd/uploads
+      - .data/CTFd/logs:/var/log/CTFd
+      - .data/CTFd/uploads:/var/uploads
+      - .:/opt/CTFd:ro
```

Comment:
How does this affect the Dockerfile COPY'ing into the image? Perhaps this should be removed?

Reply:
It will overshadow the copied in files. Doesn't effect other deployment strategies (e.g. `docker run ctfd`) but it makes it pick up changes without rebuild when deployed like this